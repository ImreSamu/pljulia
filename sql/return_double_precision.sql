CREATE FUNCTION julia_sqrt(x INTEGER)
RETURNS DOUBLE PRECISION AS $$
    sqrt(x)
$$ LANGUAGE pljulia;
SELECT julia_sqrt(5);
DROP FUNCTION julia_sqrt(x INTEGER);
