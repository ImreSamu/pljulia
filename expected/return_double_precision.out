CREATE FUNCTION julia_sqrt(x INTEGER)
RETURNS DOUBLE PRECISION AS $$
    sqrt(x)
$$ LANGUAGE pljulia;
SELECT julia_sqrt(5);
 julia_sqrt 
------------
   2.236068
(1 row)

DROP FUNCTION julia_sqrt(x INTEGER);
